src/style.css

```diff
@@ -1,3 +1,63 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
+  height: 60px;
+  width: 100%;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.links {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: rgb(0, 0, 0);
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  font-weight: 500;
+}
+
+.imagem {
+  padding: 10px 0; /* apenas pedding vertical */
+}
+
+.link.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
```

Comment:
not wrong code, but it's a good idea to add a (or more) blank line between a big list of css properties (like 6+)

just a personal suggestion